src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Square");
+        System.out.println("Color: " + getColor());
+        System.out.println("Side length: " + side);
+    }
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The `draw()` method should be in a separate interface as it serves a different purpose than `getArea()`.